internal/signalfx-agent/pkg/core/config/config.go

```diff
@@ -218,7 +218,6 @@ type CollectdConfig struct {
 	// If you won't be using any collectd monitors, this can be set to true to
 	// prevent collectd from pre-initializing
 	DisableCollectd bool `yaml:"disableCollectd" default:"false"`
-	// Deprecated: this setting has no effect and will be removed.
```

Comment:
Why are we removing the deprecated notices here?

(This might be a duplicate of @pjanotti's question, so feel free to ignore if that's the case)

Reply:
Because I made a mistake. While this code has no usage in go, it is serialized to represent collectd config on disk, and removing it breaks tests. I should not have deprecated it in the first place.